pkgs/servers/flaresolverr/default.nix

```
@@ -0,0 +1,74 @@
+{ pkgs, stdenv }:
+with pkgs;
```

Please list the dependencies explicitly.
```suggestion
{ lib, stdenv, firefox, others, here }:
```